src/doc/genbook.sh

```diff
@@ -53,7 +53,11 @@ $(cat ../apps/socket/README.md)
 
 # Libraries
 
-$(cat ../lib/README.md)
+$(cat ../lib/README.checksum.md)
```

Comment:
Oh, the header levels are off in `lib/README*`. Not sure if `pandoc` is smart about this or not.